### ISSUE TITLE
Feature: Paging

### ISSUE THREAD

**Issue by aliyss** [Sebazzz/PSMenu#10]
Allow for Paging, so items can be split up if the count exceeds X.

Navigation could then be done using the [left] and [right] arrows.

**Comment by Sebazzz** [comment]
Yes, I agree, paging would be useful and also a useful workaround until rendering performance is increased.

**Comment by aliyss** [comment]
Thinking aabout this... I had another idea of jumping in long lists.

Testing key inputs for characters and like in explorer jumping to the first output where startswith -eq InputCharacter.

Combining this with paging one could page stuff with a custom function similar to MenuFormatting and split pages with starting character -eq
